x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/generate_latest_transform.ts

```diff
@@ -52,19 +52,11 @@ export function generateLatestTransform(
         ['entity.displayName']: {
```

Comment:
since displayName can be anything, even a static value, i think it makes sense to remove it from here too